command-expansion-server/src/packages/db/db.ts

```diff
@@ -29,12 +32,12 @@ export class DbExpansion {
         user,
       };
 
-      console.log(`Postgres Config:`);
-      console.log(config);
+      logger.info(`Postgres Config:`);
+      logger.info(config);
```

Comment:
CodeQL seems to be mad at this line for leaking sensitive information, but when I look at it in `docker-compose logs -f aerie_commanding`, it seems to be printing `[Object object]`:

```
aerie_commanding    | INFO packages/db/db - Postgres Config:
aerie_commanding    | INFO packages/db/db - [object Object]
```

So... I don't think we're leaking sensitive information, but I don't think we're logging anything useful either.

Reply:
Thanks, I was just about to ask about that. Looks like CodeQL was giving that alert on the old `console.log` too.

Git says @goetzrrGit wrote that code. Ryan, is there any reason I can't delete it?

Reply:
Well, this can be done later in a little hotfix if needed. Resolving this because its not strictly part of this PR.